GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md

```diff
@@ -31,3 +31,33 @@ NativeDB Added Parameter 12: BOOL p11
 * **zAxis**: 
 
 ## Return value
+
+```
+network fx, calling this function will make the particle fx visible for all players in the server. However the 'visibility range' for other player is in certain sitations rather short-distance (in case of when using it for fireworks for example). In these situations it might be preferred to make serversided calls to play such effects locally on each client ("synced") so it would be visible 'across the map'.
+```
+
+## Example
+```lua
+-- If the PtfxAsset hasn't been loaded yet, you'll need to load it first
+if not HasNamedPtfxAssetLoaded("scr_indep_fireworks") then
+	RequestNamedPtfxAsset("scr_indep_fireworks")
+	while not HasNamedPtfxAssetLoaded("scr_indep_fireworks") do
+		Wait(10)
+	end
+end
+
+local CurrentPlayerCoords = GetEntityCoords(GetPlayerPed(-1))
+
+UseParticleFxAssetNextCall("scr_indep_fireworks") -- Prepare the Particle FX for the next upcomming Particle FX call
+SetParticleFxNonLoopedColour(1.0, 0.0, 0.0) -- Setting the color to Red (R, G, B)
+StartNetworkedParticleFxNonLoopedAtCoord("scr_indep_firework_burst_spawn", CurrentPlayerCoords, 0.0, 0.0, 0.0, 1.0, false, false, false, false) -- Start the animation itself
+
+RemoveNamedPtfxAsset("scr_indep_fireworks") -- Clean up
+```
+
+## Large Particle List:
```

Comment:
Please don't add new sections with (`##`) headers , place any information you have in the native description instead.